### PR TITLE
Feature/additional cidr

### PR DESCRIPTION
**Pull request by gayvoronskiy**
Added new CIDR range

### CODE REVIEW

File: cidr.tf [hunk 1]
```diff
@@ -0,0 +1,143 @@
+resource "aws_vpc_ipv4_cidr_block_association" "additional_cidr" {
+  for_each      = toset(var.vpc_additional_cidr)
+  vpc_id        = aws_vpc.main.id
+  cidr_block    = each.value
+}
+
+# Additional Private Subnets #
+# PRIVATE A[count+2]
```
Comment (emnavarro02) [inline cidr.tf]: Could we remove this comment. It is misleading?

File: cidr.tf [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+  tags = merge(
+    {
+      Name = local.additional_sn_private_a > 0 ? "${var.additional_sn_private_a_name}${count.index+2}" : var.additional_sn_private_a_name 
+    },
+    length(var.additional_sn_private_a_tags) != 0 ? var.additional_sn_private_a_tags[count.index] : {}
+  )
+
+  depends_on = [aws_vpc_ipv4_cidr_block_association.additional_cidr]
+}
+# PRIVATE B[count+2]
```
Comment (emnavarro02) [inline cidr.tf]: Can we remove it?

File: locals.tf [hunk 3]
```diff
@@ -88,4 +88,18 @@ locals {
...
+  additional_sn_private_b = (local.enable_dynamic_subnets == false ? 1 : (length(var.additional_private_subnets_b) > 0 ? length(var.additional_private_subnets_b) : 0))
+  additional_sn_private_c = (local.enable_dynamic_subnets == false ? 1 : (length(var.additional_private_subnets_c) > 0 ? length(var.additional_private_subnets_c) : 0))
+  additional_sn_public_a = (local.enable_dynamic_subnets == false ? 1 : (length(var.additional_public_subnets_a) > 0 ? length(var.additional_public_subnets_a) : 0))
+  additional_sn_public_b = (local.enable_dynamic_subnets == false ? 1 : (length(var.additional_public_subnets_b) > 0 ? length(var.additional_public_subnets_b) : 0))
+  additional_sn_public_c = (local.enable_dynamic_subnets == false ? 1 : (length(var.additional_public_subnets_c) > 0 ? length(var.additional_public_subnets_c) : 0))
+  # additional_private_a_tgw_attachment = local.additional_sn_private_a > 0 ? tolist([(var.tgw_attachment_aza_additional_subnet >= 0 ? aws_subnet.additional_sn_private_a[var.tgw_attachment_aza_additional_subnet].id : aws_subnet.additional_sn_private_a[0].id)]) : []
```
Comment (emnavarro02) [inline locals.tf]: Please remove it

File: variables.tf [hunk 4]
```diff
@@ -393,4 +393,118 @@ variable "nat_gw_azs" {
...
+}
+
+variable "tgw_attachment_azb_additional_subnet" {
+  description = "The private subnet in the availability zone B to be attached to the Transit Gateway"
+  default     = -1
+}
+
+variable "tgw_attachment_azc_additional_subnet" {
+  description = "The private subnet in the availability zone C to be attached to the Transit Gateway"
+  default     = -1
```
Comment (emnavarro02) [inline variables.tf]: please remove unused vars

**Review by emnavarro02 — CHANGES_REQUESTED** [review]
Hi @gayvoronskiy ,
I added a few comments. please check them out before merging

File: cidr.tf (hunk 1 above)
Comment (gayvoronskiy) [reply cidr.tf]: Comment was removed.

File: cidr.tf (hunk 2 above)
Comment (gayvoronskiy) [reply cidr.tf]: Comment was removed.

File: locals.tf (hunk 3 above)
Comment (gayvoronskiy) [reply locals.tf]: Comment was removed.

File: variables.tf (hunk 4 above)
Comment (gayvoronskiy) [reply variables.tf]: Variables deleted.

**Review by emnavarro02 — APPROVED** [review]
LGTM

**Pull request merged**